openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMUtils.java

```diff
@@ -82,6 +84,31 @@ static CpuArchitecture getCpuArchitecture(final String cpuArchName) {
         }
     }
 
+    /**
+     * Checks if the current operative system is Alpine.
+     *
+     * @return true when the operative system is Alpine and false otherwise.
+     */
+    static private boolean isAlpine() {
+        final String osReleasePath = "/etc/os-release";
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(osReleasePath));
+
+            String line;
+            while ((line = reader.readLine()) != null) {
+                if (line.equals("ID=alpine")) {
```

Comment:
How do you know alpine distributions will always have this line in /etc/os-release? Is this specified somewhere?

Reply:
I could not find a place specifying that this line should always exist. Since we must support any musl-based OS, and not only Alpine, this code was removed, and now the detection is based of an env variable.